tests/test_kubernetes_operator.py

```diff
@@ -19,13 +19,13 @@ def dag(self):
 
     def test_that_exception_is_raised_for_empty_image(self, dag):
         with pytest.raises(MissingValueException) as info:
-            kubernetes_operator(dag, "name", "repo", "")
+            kubernetes_operator(dag, "name", "", "repo")
```

Comment:
```suggestion
            kubernetes_operator(dag, "name", "image", "repo")
```

Reply:
Testen er for å sjekke at exception blir kastet dersom man utelater image.